qt-models/divelistmodel.cpp

```diff
@@ -218,6 +217,12 @@ int DiveListModel::getDiveIdx(int id) const
 	return get_idx_by_uniq_id(id);
 }
 
+QModelIndex DiveListModel::getDiveQIdx(int id)
+{
+	int idx = getDiveIdx(id);
+	return idx >= 0 ? createIndex(idx, 0) : QModelIndex();
```

Comment:
I read this and the block above for `getIdxForId` three times and they still confuse me. Could you add a couple of comments that help the slow learners what this does? 
`getDiveQIdx` vs `getDiveIdx`?

Reply:
One returns an integer, one a QModelndex. Will add comments.

Reply:
Added comments. Not sure if they are all that helpful though.